tools/prove_wrapper

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# shellcheck disable=SC2048,SC2086
+
+echo "Running prove with TAP output check"
+OUTPUT=$(prove "$@" 2>&1)
+STATUS=$?
+
+echo "Output:"
+echo "$OUTPUT"
+
+if [ "$STATUS" -ne 0 ]; then
+    echo "Failed with status $STATUS"
+    exit $STATUS
+fi
```

Comment:
As @perlpunk and me suggested this looks like you want to use `tee` instead. Also probably it's not a good idea to do `2>&1` because I assume one of the two is already properly handled? Also consider [Test::Warnings here](https://metacpan.org/pod/Test::Warnings)

Reply:
> Well, it catches warnings so I assume it tapped into `warn`.

Test::Warnings set `$SIG{__WARN__}`, but it does not override the `warn` function in any way.

> Maybe we can tap into "usual output" functions like "printf"? and catch output this way?

How do you want to differentiate between print statements we need and those which we don't need?
I don't think that's going to work.

Reply:
> I found https://metacpan.org/pod/Capture::Tiny#tee which looks like it could be useful

Test::Output is using Capture::Tiny behind the scenes.

Reply:
I'm not sure what both of you are talking about, but just for context i need to say that I've stopped, not even push, the experiment I was doing using perl to do this by now. Do you think I need to switch to it? I think is not necessary due to the not that complexity of the check we need, but critiques are welcome.